### ISSUE TITLE
Problem connecting to WiFi

### ISSUE THREAD

**Issue by enwol** [leoclee/7-segment-clock#7]
Hi!
I made one of this clock earlier, and it worked perfect! I have now made my second one and tried to use two leds per segment, but when I try to connect this one to the WiFi I get this output on the Serial Monitor. Does anyone know what the problem can be?

*WM: Sent wifi save page
*WM: Connecting to new AP
*WM: Connecting as wifi client...
*WM: Status:
*WM: 0
*WM: [ERROR] WiFi.begin res:
*WM: 6
*WM: Connection result: 
*WM: 4
*WM: Failed to connect.
*WM: Request redirected to captive portal
*WM: Request redirected to captive portal

**Comment by leoclee** [comment]
This looks like [WiFiManager](https://github.com/tzapu/WiFiManager) is unable to connect to your wifi for some reason, but there could be many reasons why this is happening. One thing you can try is to upload WifiManager's basic example at https://github.com/tzapu/WiFiManager/blob/master/examples/Basic/Basic.ino to your D1 mini and ensure that is able to connect once you have entered in the proper SSID and password--this would at least eliminate the possibility of it being something caused by the clock code.

**Comment by enwol** [comment]
Thank you for answering! I have now tried that, but I get the same output on the Serial Monitor.. My computer got updated to Windows 11 about a month ago, can that be the problem?

**Comment by leoclee** [comment]
I don't believe windows 11 would have an effect on this. Do you have another D1 mini to test with? Could be a hardware issue. Or it could be a network issue--are other new devices are able to join the wifi using the same SSID/password? The D1 mini can only connect on 2.4ghz; sometimes it's possible to have a different password set up on your router for 2.4 vs 5ghz.

**Comment by enwol** [comment]
I have ordered a new D1 mini now, and hopefully thats where the problem is. Other devices have no problem connecting to the same SSID/password and I'm sure it is 2.5 ghz
